CodePush.xcodeproj/project.pbxproj

```diff
@@ -215,9 +215,8 @@
 				HEADER_SEARCH_PATHS = (
 					"$(inherited)",
 					/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/include,
-					"$(SRCROOT)/../../React/**",
-					"$(SRCROOT)/../../node_modules/react-native/React/**",
 					"$(SRCROOT)/node_modules/react-native/React/**",
+					"$(SRCROOT)/Examples/CodePushDemoApp/node_modules/react-native/React/**",
```

Comment:
The deleted lines were wrong I think. $(SRCROOT) in the CodePush project is relative to itself. Having "$(SRCROOT)/node_modules/react-native/React/**" is sufficient. The added line supports loading from the Example project's node_modules in the case where the example project is the source root.